contracts/linear/src/account.rs

```diff
@@ -9,7 +9,7 @@ use std::collections::HashMap;
 use crate::types::*;
 
 /// Inner account data of a delegate.
-#[derive(BorshDeserialize, BorshSerialize, Debug, PartialEq, Default)]
+#[derive(BorshDeserialize, BorshSerialize, Debug, PartialEq, Eq, Default)]
```

Comment:
why is `Eq` necessary

Reply:
https://rust-lang.github.io/rust-clippy/master/#derive_partial_eq_without_eq